### PR TITLE
Hotfix elim rotation

### PR DESCRIPTION
**Pull request by takemi853**
This patch fix is the bug on elimination target pointing.

### CODE REVIEW

File: assignment.py [hunk 1]
```diff
@@ -107,9 +107,9 @@ def elim_rotation(cand_shortlists, reviewer_shortlists, rotation):
     rotation_tuples = [(c, cand_shortlists[c][0]) for c in rotation]
     removals = []
 
-    for c, r in rotation_tuples:
-        # Remove p (anyone same or worse than c) from r's list
-        for _ in range(len(reviewer_shortlists[r]) - reviewer_shortlists[r].index(c)):
+    for i, (c, r) in enumerate(rotation_tuples):
+        # Remove p (anyone same or worse than rotaion[i]) from r's list
```
Comment (maxkapur) [inline assignment.py]:
```suggestion
        # Remove p (anyone same or worse than rotation[i]) from r's list
```

**Review by maxkapur — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a few minor issues:
- Please remove the cached binary files `__pycache__/maxflow.cpython-39.pyc` and `__pycache__/maxflow.cpython-39.pyc` (I should have put these in `.gitignore` at the outset)
- Small typo `rotaion` in a comment

**Comment by takemi853** [comment]
I removed the files and fixed the typo!

**Comment by maxkapur** [comment]
Looks great, thank you!

**Pull request merged**